lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -436,19 +438,25 @@ sub run_backup {
             format => 'HTML',
         );
         return $template->render_to_psgi($request);
-    } elsif ($request->{backup_type} eq 'browser') {
+    }
+    elsif ($request->{backup_type} eq 'browser') {
         my $bak;
         open $bak, '<', $backupfile
             or die "Failed to open temporary backup file $backupfile : $!";
         unlink $backupfile; # remove the file after it gets closed
 
         my $attachment_name = 'ledgersmb-backup-' . time . '.sqlc';
```

Comment:
Not certain what format `time` is here, but it should match the timestamp used in the filename above.
Probably `yymmdd-hhMMss`